tbx/core/templatetags/util_tags.py

```diff
@@ -88,3 +89,24 @@ def ifinlist(value, list):
     # cast to strings before testing as this is used for heading levels 2, 3, 4 etc
     stringList = [str(x) for x in list]
     return str(value) in stringList
+
+
+@register.filter(name="has_raw_html_block")
```

Comment:
suggestion: rename this to `has_gist_block` - because you can still have an html block without gist and this would return false.

Reply:
Good shout, i'll update